Source/Tests/LinuxSyscalls/Syscalls.cpp

```diff
@@ -268,8 +268,8 @@ uint64_t ExecveHandler(const char *pathname, char* const* argv, char* const* env
       Filename = pathname;
     }
 
-    bool exists = std::filesystem::exists(Filename, ec);
-    if (ec || !exists) {
+    bool exists = FHU::Filesystem::Exists(Filename.c_str());
```

Comment:
Could we add a dedicated overload for `Exists(fextl::string)`? Seems to come up a couple of times throughout the PR.